packages/generate/src/syntax/toEdgeQL.ts

```diff
@@ -473,16 +480,9 @@ function walkExprTree(
     case ExpressionKind.PathLeaf:
     case ExpressionKind.PathNode:
       if (expr.__parent__) {
-        if ((expr.__parent__.type as any).__scopedFrom__) {
```

Comment:
Are we losing this special check or is the `expr.__scopedFrom__` going to catch the same things that this `expr.__parent__.type.__scopedFrom__` check is looking for?

Reply:
Yes, the scoped expr check for `expr.__parent__.type` is now just inside the `walkExprTree` call, so it also catches the scoped exprs wherever they're used, and not just in paths.

Reply:
ahh, that makes sense 👍